develop-guide/writeblitz/writeblitz.rst

```diff
@@ -751,6 +811,7 @@ Trigger timeout/interval ratio adjustments
 
 Each action performs verification to make sure it has performed as expected.
 These timeouts can be modified with a ratio from the testbed datafile.
+Actions ``api``, ``execute``, ``parse``, ``learn`` and ``rest`` are supporting this feature
```

Comment:
This feature is supported by actions `api`,`execute`, `parse`, `learn` and `rest`.

above is better?

Reply:
DOne